src/client/src/orangehrmPimPlugin/components/FullNameInput.vue

```diff
@@ -80,6 +80,10 @@ export default {
       type: Boolean,
       default: true,
     },
+    label: {
```

Comment:
No need for label here? we can remove `inheritAttrs: false` then the label will inherit without props